extensions/wikia/DesignSystem/controllers/DesignSystemGlobalFooterController.class.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class DesignSystemGlobalFooterController extends WikiaController {
+	public function index() {
+		$this->setVal( 'model', $this->getData() );
+	}
+
+	public function section() {
```

Comment:
We'll handle escaping data in XW-1729. We're going to change templates a little bit and add translations there.